network/src/peer/test.rs

```diff
@@ -749,7 +859,11 @@ fn peer_disconnect_connection_lost() {
     ::diem_logger::Logger::init_for_testing();
     let rt = Runtime::new().unwrap();
     let (peer, _peer_handle, mut connection, mut connection_notifs_rx, _peer_notifs_rx) =
-        build_test_peer(rt.handle().clone(), ConnectionOrigin::Inbound);
+        build_test_peer(
+            rt.handle().clone(),
+            TimeService::mock(),
+            ConnectionOrigin::Inbound,
+        );
```

Comment:
You could probably just stick `TimeService::mock()` in `build_test_peer`, unless it's used otherwise.

Ah, I see in one test you pass in a MockTimeService.  It's times like these that I wish Rust had function overloading, so we could just simply have the same name, but ignore the `TimeService` field.  Similarly with the `ConnectionOrigin`, doesn't look like it changes at all (probably my fault)